src/app/domain/backup/services/manageBackup.ts

```diff
@@ -124,6 +124,20 @@ export const startBackup = async (
           postUploadLocalBackupConfig.currentBackup.totalMediasToBackupCount,
         message: partialSuccessMessage
       })
+
+      await showLocalNotification({
```

Comment:
nit: The only difference between `if` and `else` blocks are the status, and the title/body translation keys. So you can ease identifying differences by:

```js
let status = 'success'
let titleKey = 'services.backup.notifications.backupSuccessTitle'
let bodyKey = 'services.backup.notifications.backupSuccessBody'

if (partialSuccessMessage) {
  status = 'partial_success'
  titleKey = 'services.backup.notifications.backupPartialSuccessTitle'
  bodyKey = 'services.backup.notifications.backupPartialSuccessBody'
}

await setLastBackup(...)

await showLocalNotification(...)
```